test/lit/basic/typed_continuations.wast

```diff
@@ -38,7 +38,7 @@
   ;; CHECK-TEXT:      (func $id2 (type $3) (param $w contref) (param $x nullcontref) (param $y (ref cont)) (param $z (ref nocont)) (result contref)
   ;; CHECK-TEXT-NEXT:  (local.get $z)
   ;; CHECK-TEXT-NEXT: )
-  ;; CHECK-BIN:      (func $id2 (type $3) (param $w contref) (param $x nullcontref) (param $y (ref func)) (param $z (ref nocont)) (result contref)
+  ;; CHECK-BIN:      (func $id2 (type $3) (param $w contref) (param $x nullcontref) (param $y (ref cont)) (param $z (ref nocont)) (result contref)
```

Comment:
Was this just wrong before? It does look right now. What fixed it?

Reply:
Hehe, yes, this was just wrong: https://github.com/WebAssembly/binaryen/pull/6667/files#diff-666d70a4550a2ccec6247c639d54231a05108efc40af830403dfbc6527b251f7R2087-R2088

Reply:
I missed that on my read, good catch.